services/powchain/src/main/java/tech/pegasys/teku/services/powchain/PowchainService.java

```diff
@@ -62,11 +66,19 @@ public PowchainService(final ServiceConfig config) {
 
     this.web3j = createWeb3j(tekuConfig);
 
+    // TODO init list with multiple endpoints from CLI options
+    final List<Eth1Provider> eth1Providers =
+        Arrays.asList(
+            new ThrottlingEth1Provider(
+                new ErrorTrackingEth1Provider(
```

Comment:
We will probably have to adjust where the `ErrorTrackingEth1Provider` is put in this chain.  It may wind up needing to be aware of the multiple providers so it can provide a warning that the primary eth1 node is unavailable but backup is working vs all providers being down etc.  Possibly the functionality just moves into `FallbackAwareEth1Provider` - not sure exactly.

Reply:
Done. The `ErrorTrackingEth1Provider` now encapsulates the `FallbackAwareEth1Provider`